script.js

```diff
@@ -0,0 +1,74 @@
+let count = 0;
+
+let firstCard = "";
+
+let secondCard = "";
+
+const gameSpaces = document.querySelectorAll(".gameSpace");
+
+gameSpaces.forEach((gameSpace) => {
+  gameSpace.addEventListener("click", showCard);
+});
+
+function toggleClicks() {
+  gameSpaces.forEach((gameSpace) => {
+    gameSpace.classList.toggle("preventClick");
+  });
+}
+
+function clearCards(event) {
+  firstCard.innerHTML = "";
+  secondCard.innerHTML = "";
+  count = 0;
+  firstCard.disabled = false;
+  secondCard.disabled = false;
+  firstCard = "";
+  secondCard = "";
+  toggleClicks();
+}
+
+function showCard(event) {
+  count++;
+  if (event.target.classList.contains("1")) {
+    event.target.innerHTML = "!";
+  }
+  if (event.target.classList.contains("2")) {
+    event.target.innerHTML = "@";
+  }
+  if (event.target.classList.contains("3")) {
+    event.target.innerHTML = "#";
+  }
+  if (event.target.classList.contains("4")) {
+    event.target.innerHTML = "$";
+  }
+  if (event.target.classList.contains("5")) {
+    event.target.innerHTML = "%";
+  }
+  if (event.target.classList.contains("6")) {
+    event.target.innerHTML = "^";
+  }
+  if (event.target.classList.contains("7")) {
+    event.target.innerHTML = "&";
+  }
+  if (event.target.classList.contains("8")) {
```

Comment:
you may want to change this `if` and all the above as well until line 32 into `else if`
since you are checking if it is one of the choices and not if it is any of the choices.

You could also replace this with a `switch` case https://www.w3schools.com/js/js_switch.asp